### PR TITLE
7770 add footer

### PR DESCRIPTION
**Pull request by spavelo**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -211,7 +211,38 @@
...
         </article>
+        <footer>
+            <div class="footer_container">
+                <div>
+                    <div class="category small_fill_1">корисні посилання</div>
+                    <button class="small_white">Відділення та банкомати</button>
+                    <button class="small_white">Оплата комунальних послуг</button>
+                    <button class="small_white">Поповнення мобільного</button>
+                    <button class="small_white">Контакти</button>
+                    <button class="small_white">Мобільні додатки</button>
```
Comment (SVladikO) [inline src/index.html]: use scss @extend, @incluede

File: src/index.html [hunk 2]
```diff
@@ -211,7 +211,38 @@
...
+                    </div>
+                    <div class="medium_x_white">+380 44 363-01-33</div>
+                </div>
+                <div>
+                    <div class="category small_fill_1">ОЩАДБАНК у соціальних мережах</div>
+                    <div class="social_media">
+                        <a href="/img/footer/f.png"></a>
+                        <a href="/img/footer/ph.png"></a>
+                        <a href="/img/footer/tw.png"></a>
+                        <a href="/img/footer/yt.png"></a>
```
Comment (SVladikO) [inline src/index.html]: use appropriate names

File: src/style/abstracts/_mixin_implementation.scss [hunk 3]
```diff
@@ -26,15 +26,17 @@
 .large_primary {
   @include font_fabric($font_large, $text_color_fill_3);
 }
+.medium_x_white {
```
Comment (SVladikO) [inline src/style/abstracts/_mixin_implementation.scss]: rename on x_medium_white
code consistency

File: src/style/abstracts/_mixin_implementation.scss [hunk 4]
```diff
@@ -26,15 +26,17 @@
...
 .large_x_white {
   @include font_fabric($font_x_large, $text_color_white);
-
 }
 .large_xxl_primary {
   @include font_fabric($font_xxl_large, $text_color_white);
   font-family: $font_third;
 }
-.large_xxl_medium {
+.large_xxl_white {
```
Comment (SVladikO) [inline src/style/abstracts/_mixin_implementation.scss]: name consistency

File: src/style/abstracts/_variables.scss [hunk 5]
```diff
@@ -9,16 +9,20 @@ $bg_color_fill_1: #929c9f;
 $bg_color_fill_2: #dcdf3c;
 $bg_color_fill_3: #7196da;
 $bg_color_fill_4: #212628;
+$bg_color_fill_5:#4B5153;
 $bg_color_white: #fff;
 $bg_color_black: #000;
 
 $text_color_fill_1: #929c9f;
+$text_color_fill_2: #585b5c;
```
Comment (SVladikO) [inline src/style/abstracts/_variables.scss]: fill_2 for text_color and for bg_color should have the same value

File: src/style/layout/_footer.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+      justify-content: space-around;
+      flex-direction: column;
+      padding: 8px 0;
+
+      button {
+        background: $bg_color_fill_4;
+        border: none;
+        text-align: start;
+        padding: 5px 0;
+        @include interaction(#ebebeb, #C2C52E, #A8AA2A, #DCE123);
```
Comment (SVladikO) [inline src/style/layout/_footer.scss]: use color variables instead

File: src/style/layout/_footer.scss [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+    .social_media {
+      display: flex;
+      flex-direction: row;
+      justify-content: start;
+
+      a {
+        height: 18px;
+        width: 18px;
+        border: 1px solid red;
```
Comment (SVladikO) [inline src/style/layout/_footer.scss]: red ?

File: src/style/layout/_footer.scss [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+        margin: 0 5px 0 0;
+      }
+    }
+
+    .medium_x_white {
+      font-family: $font_fourth;
```
Comment (SVladikO) [inline src/style/layout/_footer.scss]: why medium_x_white without font_factory?

File: src/style/layout/_header.scss [hunk 9]
```diff
@@ -18,7 +18,7 @@ header {
   background: $bg_color_black;
   border: none;
   color: $color_fill_1;
-  @include name($color_fill_1, #C2C52E, #A8AA2A, #DCE123);
+  @include interaction($color_fill_1, #C2C52E, #A8AA2A, #DCE123);
```
Comment (SVladikO) [inline src/style/layout/_header.scss]: use color variables

File: src/style/layout/_header.scss [hunk 10]
```diff
@@ -36,7 +36,7 @@ header > button {
 }
 
 .type_of_business > button {
-  @include name($color_fill_1, #C2C52E, #A8AA2A, #DCE123);
+  @include interaction($color_fill_1, #C2C52E, #A8AA2A, #DCE123);
```
Comment (SVladikO) [inline src/style/layout/_header.scss]: use color variables.

File: src/style/abstracts/_variables.scss [hunk 11]
```diff
@@ -9,16 +9,20 @@ $bg_color_fill_1: #929c9f;
...
 $text_color_fill_3: #7196da;
 $text_color_fill_4: #212628;
+$text_color_fill_5: #545759;
+
 $text_color_white: #fff;
 $text_color_black: #000;
 
-$font_small: 9px;
+$font_small: 11px;
```
Comment (SVladikO) [inline src/style/abstracts/_variables.scss]: I don't like this solution.

Comment (SVladikO) [reply src/style/abstracts/_variables.scss]: Are sure that we didn't brake any code ?

File: src/style/abstracts/_variables.scss [hunk 12]
```diff
@@ -9,16 +9,20 @@ $bg_color_fill_1: #929c9f;
 $bg_color_fill_2: #dcdf3c;
 $bg_color_fill_3: #7196da;
 $bg_color_fill_4: #212628;
+$bg_color_fill_5:#4B5153;
```
Comment (SVladikO) [inline src/style/abstracts/_variables.scss]: put space before #

File: src/index.html (hunk 1 above)
Comment (SVladikO) [reply src/index.html]: use  footer_container > div > button.small_white

Comment (spavelo) [reply src/index.html]: done

File: src/index.html (hunk 2 above)
Comment (spavelo) [reply src/index.html]: DONE

File: src/style/abstracts/_mixin_implementation.scss (hunk 3 above)
Comment (spavelo) [reply src/style/abstracts/_mixin_implementation.scss]: done

File: src/style/abstracts/_mixin_implementation.scss (hunk 4 above)
Comment (spavelo) [reply src/style/abstracts/_mixin_implementation.scss]: DONE

File: src/style/layout/_footer.scss (hunk 6 above)
Comment (spavelo) [reply src/style/layout/_footer.scss]: DONE

File: src/style/layout/_footer.scss (hunk 8 above)
Comment (spavelo) [reply src/style/layout/_footer.scss]: DONE

File: src/style/layout/_footer.scss (hunk 7 above)
Comment (spavelo) [reply src/style/layout/_footer.scss]: DONE

File: src/style/layout/_header.scss (hunk 9 above)
Comment (spavelo) [reply src/style/layout/_header.scss]: DONE

File: src/style/layout/_header.scss (hunk 10 above)
Comment (spavelo) [reply src/style/layout/_header.scss]: DONE

File: src/style/abstracts/_variables.scss (hunk 11 above)
Comment (spavelo) [reply src/style/abstracts/_variables.scss]: Yes, im sure

File: src/style/abstracts/_variables.scss (hunk 12 above)
Comment (spavelo) [reply src/style/abstracts/_variables.scss]: DONE

**Pull request merged**